rn-cli.config.js

```diff
@@ -0,0 +1,11 @@
+var blacklist = require('react-native/packager/blacklist')
+
+var config = {
+  getBlacklistRE(platform) {
+    return blacklist(platform, [
+      /coverage\/.*/
+    ])
+  }
+}
+
+module.exports = config
```

Comment:
if you run tests with coverage, react native tries to transpire the reports 👯‍♂️